client/a8c-for-agencies/sections/purchases/payment-methods/payment-method-overview/index.tsx

```diff
@@ -10,39 +12,108 @@ import LayoutHeader, {
 import LayoutTop from 'calypso/a8c-for-agencies/components/layout/top';
 import MobileSidebarNavigation from 'calypso/a8c-for-agencies/components/sidebar/mobile-sidebar-navigation';
 import { A4A_PAYMENT_METHODS_ADD_LINK } from 'calypso/a8c-for-agencies/components/sidebar-menu/lib/constants';
+import Pagination from 'calypso/components/pagination';
+import { PaymentMethod } from 'calypso/jetpack-cloud/sections/partner-portal/payment-methods';
+import StoredCreditCardV2 from 'calypso/jetpack-cloud/sections/partner-portal/stored-credit-card-v2';
```

Comment:
To avoid bloating this PR, we are reusing the StoredCreditCard component from Jetpack Manage for now. This will be addressed on a separate PR.